src/main/kotlin/upbrella/be/store/service/StoreImageService.kt

```
@@ -75,6 +91,66 @@ class StoreImageService(
+
+            return ImageUrlsResponse(
+                id = null,
+                webp = ImageSizeUrls(thumbWebp, mediumWebp, largeWebp),
+                jpeg = ImageSizeUrls(thumbJpeg, mediumJpeg, largeJpeg)
+            )
+        } catch (e: Exception) {
+            // 업로드 실패 시 이미 업로드된 파일들 삭제 (rollback)
+            uploadedKeys.forEach { url ->
+                try {
+                    deleteFileInS3(url)
+                } catch (deleteError: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)